recipe/conda_build_config.yaml

```diff
@@ -1,21 +1,28 @@
-# We must use gcc version 8:
+# We must use gcc version 8 with cuda <11:
 #   cuda_runtime.h:83: cudagcc versions later than 8 are not supported!
-
 c_compiler:
   - gcc                        # [linux]
 c_compiler_version:            # [unix]
-  - 8                          # [linux64]
+  - 9                          # [linux64]
 cxx_compiler:
   - gxx                        # [linux]
 cxx_compiler_version:          # [unix]
-  - 8                          # [linux64]
+  - 9                          # [linux64]
 cgo_compiler:
   - go-cgo
+cgo_compiler_version:          # [unix]
+  - 1.15                       # [linux64]
 cuda_compiler:
   - nvcc
 cuda_compiler_version:
   - None
-  - 10.2                       # [linux64]
+  - 11.0                       # [linux64]
 pin_run_as_build:
   boost-cpp:
     max_pin: x.x.x
+cdt_name:                      # [linux64]
+  - cos7                       # [linux64]
+  - cos7                       # [linux64]
```

Comment:
do note that this breaks compatibility with a lot of older packages (or, rather, older packages start to be incompatible with OmniSci after this), what's the reason for changing?

Reply:
omnisci switches to requiring c++17 and `cuda>=11` that also means `gcc>=9`.

I can try removing `cdt_name` to ensure that this change is still relevant.

Reply:
Confirmed, `cdt_name` is not needed and removed now.